docs/config.md

```diff
@@ -176,13 +176,35 @@ diff-args = ["--color=always", "$left", "$right"]
 - `$left` and `$right` are replaced with the paths to the left and right
   directories to diff respectively.
 
+### Set of immutable commits
+
+You can configure the set of immutable commits via `revsets.immutable-heads`.
+For example, to prevent accidentally rewriting commits on `main@origin`, use:
+
+```toml
+revsets.immutable-heads = "main@origin"
+```
+
+To prevent rewriting commits on `main@origin` and commits authored by other
+users:
+
+```toml
+# The `main.. &` bit is an optimization to scan for non-`mine()` commits only
+# among commits that are not in `main`.
+revsets.immutable-heads = "main@origin | (main@origin.. & ~mine())"
+```
+
+Ancestors of the configured set are also immutable. The root commit always
+immutable even if the set is empty. The resulting set of immutable commits is
```

Comment:
I think you should mention the default somewhere. E.g. "...even if the set is empty (which is the default)".

Reply:
Done. I also dropped the `main@origin` example because that's now basically the default since we use `trunk()`